platform/web/tests/decode_with_json_schema_test.go

```diff
@@ -10,190 +11,161 @@ import (
-			Body:           `{"code": 5, "datatype": "integer"}`,
-			JSONSchemaPath: "../../../testdata/json-schema/json_schema_with_definition.json",
-			DecoderOptions: web.DecoderOptions{},
-			ExpectedError: web.NewErrBadRequestResponse(web.ErrorDetails{
-				"id":   "id is required",
-				"code": "Invalid type. Expected: string, given: integer",
-			}),
-		},
-		{
-			Name:           "It can properly validate the localized string",
-			Body:           `{"code": "code", "id": "d43c45b0-f420-4de9-8745-6e3840ab39fd", "datatype": "integer", "name": {"en-US": "this is a test", "fr-FR": "another test"}}`,
-			JSONSchemaPath: "../../../testdata/json-schema/json_schema_with_definition.json",
-			DecoderOptions: web.DecoderOptions{},
-			ExpectedError:  nil,
-			ExpectedOutcome: expectedModel{
-				ID:       "d43c45b0-f420-4de9-8745-6e3840ab39fd",
-				Code:     "code",
-				DataType: "integer",
-				Name:     map[string]string{"en-US": "this is a test", "fr-FR": "another test"},
-			},
-		},
-		{
-			Name:           "It throws an error when a required nested property is missing",
-			Body:           `{"code": "code", "id": "d43c45b0-f420-4de9-8745-6e3840ab39fd", "datatype": "integer", "name": {"fr-FR": "another test"}}`,
-			JSONSchemaPath: "../../../testdata/json-schema/json_schema_with_definition.json",
-			DecoderOptions: web.DecoderOptions{},
-			ExpectedError: web.NewErrBadRequestResponse(web.ErrorDetails{
-				"name": "en-US is required",
-			}),
-		},
-		{
-			Name:           "It throws an error when a required nested property is missing",
-			Body:           `{"code": "code", "id": "d43c45b0-f420-4de9-8745-6e3840ab39fd", "datatype": "integer", "name": {"en-US": "this is a test", "French": "ceci est une test"}}`,
-			JSONSchemaPath: "../../../testdata/json-schema/json_schema_with_definition.json",
-			DecoderOptions: web.DecoderOptions{},
-			ExpectedError: web.NewErrInvalidRequestBodyContent(web.ErrorDetails{
-				"name": "Additional property French is not allowed",
-			}),
-		},
-	}
+	},
+	{
+		Name:           "It throws an error when a required nested property is missing",
+		Body:           `{"code": "code", "id": "d43c45b0-f420-4de9-8745-6e3840ab39fd", "datatype": "integer", "name": {"fr-FR": "another test"}}`,
+		DecoderOptions: web.DecoderOptions{},
+		ExpectedError: web.NewErrBadRequestResponse(web.ErrorDetails{
+			"name": "en-US is required",
+		}),
+	},
+	{
+		Name:           "It throws an error when another required nested property is missing",
+		Body:           `{"code": "code", "id": "d43c45b0-f420-4de9-8745-6e3840ab39fd", "datatype": "integer", "name": {"en-US": "this is a test", "French": "ceci est une test"}}`,
+		DecoderOptions: web.DecoderOptions{},
+		ExpectedError: web.NewErrInvalidRequestBodyContent(web.ErrorDetails{
+			"name": "Additional property French is not allowed",
+		}),
+	},
+}
+
+//go:embed testdata/json_schema_with_definition.json
+var jsonSchema []byte
```

Comment:
I think this one can go in the function just below

Reply:
We can't use embed inside of functions or this happens:

```
# github.com/fewlinesco/go-pkg/platform/web/tests [github.com/fewlinesco/go-pkg/platform/web/tests.test]
./decode_with_json_schema_test.go:136:4: go:embed cannot apply to var inside func
```

I just found out trying to do so 😅

Reply:
Okay :) I guessed it was a possibility